.circleci/config.yml

```diff
@@ -19,13 +19,14 @@ jobs:
           name: Run unit tests for the front end
           command: |
             set -x
-            docker-compose run frontend npm test
+            npm install
+            npm test
       - run:
           name: Code coverage
           command: |
             set -x
             echo CODECOV_TOKEN=$CODECOV_TOKEN >> .env
-            docker-compose run frontend npm run coverage
+            NODE_ENV=test npm install -g codecov && codecov
```

Comment:
Running in Docker containers was causing the heap sizes of our tests to grow too large and crash the test runner. This is not a permanent solution.